qa/pull-tester/rpc-tests.sh

```diff
@@ -34,6 +34,7 @@ testScripts=(
     'maxblocksinflight.py'
     'invalidblockrequest.py'
     'rawtransactions.py'
+    'retarget-overflow.py'
```

Comment:
IMO best practice (for the current state) is to add your test commented out in `rpc-tests.sh`.